lib/koans/05_maps.ex

```diff
@@ -43,7 +43,7 @@ defmodule Maps do
 
   koan "Can remove pairs by key" do
     without_age = Map.delete(@person, :age)
-    assert Map.keys(without_age) |> Enum.sort == ___
+    assert Enum.sort(Map.keys(without_age)) == ___
```

Comment:
Maybe we could replace this with [Map.has_key?/2](http://elixir-lang.org/docs/stable/elixir/Map.html#has_key?/2) so that we don't rely on the key order? I'm thinking something like:

```elixir
assert Map.has_key?(without_age, :age) == false
```